src/test/scala/lasp/hapi/util/TimeSpec.scala

```diff
@@ -0,0 +1,74 @@
+package lasp.hapi.util
+
+import java.time.LocalDateTime
+
+import org.scalactic.Equality
+import org.scalatest.Assertion
+import org.scalatest.FlatSpec
+
+class TimeSpec extends FlatSpec {
+
+  val expected = LocalDateTime.of(2018, 1, 1, 0, 0, 0)
+
+  def testParse(str: String): Assertion =
+    assert {
+      expected === Time.parse(str).getOrElse {
```

Comment:
I gather that `===` uses the `Equality` type class?

Reply:
That's correct. Because of how `LocalDateTime` works, we can't reliably use `==` or anything hard-coded to use `==` (like `assertResult`), so we get to define an `Equality` instance and use `===`.